src/AWS.jl

```diff
@@ -44,6 +44,23 @@ using ..AWSExceptions: AWSException
 const user_agent = Ref("AWS.jl/1.0.0")
 const aws_config = Ref{AbstractAWSConfig}()
 
+function amazonify(mapping, kwargs)
+    new_kwargs = Dict{String,Any}()
+
+    for kwarg in kwargs
+        str = string(kwarg.first)
+        _key = if haskey(mapping, str)
+            mapping[str]
+        else
+            str
+        end
+
+        new_kwargs[string(_key)] = kwarg.second
```

Comment:
I don't think  `(::Pair).first` and `(::Pair).second` is part of the public API -- fields generally are not.
Would it be clearer if we used iterator destructuring to put them in variables?
idk if `k` and `v` are the best names -- all the names here are unclear to me.
I don't really now what this function is doing.

Also perhaps clearer tpo use `get` with default
```suggestion
    for (k, v) in kwargs
        str = string(k)
        _key = get(mapping, str, str)
        new_kwargs[string(_key)] = v
```